app/controllers/hyrax/admin/permission_template_accesses_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# OVERRIDE FILE from Hryax v2.9.0
```

Comment:
- correct format
- fix spelling of Hyrax
- make sure override upgrade to Hyrax 3.4